docs/docs/theme-ui.md

```diff
@@ -0,0 +1,264 @@
+  colors: {
+    text: "#333",
+    background: "#fff",
+    primary: "#639",
+    secondary: "#ff6347",
+  },
+}
+```
+
+Next add some base typographic values.
+
+```js:title=src/gatsby-plugin-theme-ui/index.js
+export default {
+  colors: {
+    text: "#333",
+    background: "#fff",
+    primary: "#639",
+    secondary: "#ff6347",
+  },
+  // highlight-start
+  fonts: {
+    body: "system-ui, sans-serif",
+    heading: "system-ui, sans-serif",
+    monospace: "Menlo, monospace",
+  },
+  fontWeights: {
+    body: 400,
+    heading: 700,
+    bold: 700,
+  },
+  lineHeights: {
+    body: 1.5,
+    heading: 1.125,
+  },
+  fontSizes: [12, 14, 16, 20, 24, 32, 48, 64, 72],
+  // highlight-end
+}
+```
+
+Next add values for use in margin and padding.
+
+```js:title=src/gatsby-plugin-theme-ui/index.js
+export default {
+  colors: {
+    text: "#333",
+    background: "#fff",
+    primary: "#639",
+    secondary: "#ff6347",
+  },
+  fonts: {
+    body: "system-ui, sans-serif",
+    heading: "system-ui, sans-serif",
+    monospace: "Menlo, monospace",
+  },
+  fontWeights: {
+    body: 400,
+    heading: 700,
+    bold: 700,
+  },
+  lineHeights: {
+    body: 1.5,
+    heading: 1.125,
+  },
+  fontSizes: [12, 14, 16, 20, 24, 32, 48, 64, 72],
+  // highlight-start
+  space: [0, 4, 8, 16, 32, 64, 128, 256, 512],
+  // highlight-end
+}
+```
+
+Feel free to include as many additional values as you'd like in your theme.
+Read more about theming in the [Theme UI documentation](https://theme-ui.com/theming).
+
+## Adding styles to elements
+
+Theme UI uses a custom [JSX pragma][] comment to add support for Emotion's `css` prop,
+which can be used to style elements.
+By default JSX compiles to the `React.createElement` function.
+When you add the `/** @jsx jsx */` comment to the top of a file, JSX will compile to the function name in the comment instead.
+For example, `<button />` would compile to `jsx('button')`.
+Theme UI's `jsx` function still renders React elements but lets you use the `css` prop for styling.
+
+[jsx pragma]: https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#pragma
+
+The following example uses the `css` prop to add styles to an element.
+
+```js:title=src/components/header.js
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+
+export default props => (
+  <header
+    css={{
+      padding: 32,
+      color: "#fff",
+      backgroundColor: "#639",
+    }}
+  >
+    {props.children}
+  </header>
+)
+```
+
+With Emotion's API, values from the theming context can be referenced by passing a function to the `css` prop.
+
+```js:title=src/components/header.js
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+
+export default props => (
+  <header
+    css={theme => ({
+      padding: theme.space[4],
+      color: theme.colors.background,
+      backgroundColor: theme.colors.primary,
+    })}
+  >
+    {props.children}
+  </header>
+)
+```
+
+Theme UI includes the [`sx` prop][], which can be used as a replacement for the `css` prop.
```

Comment:
This section could be another advanced note like I mentioned above.

Reply:
With https://github.com/gatsbyjs/gatsby/pull/15764/files#r305564999, this should be fine as-is!